airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorMetadataDefinitionV0.yaml

```diff
@@ -113,3 +113,5 @@ properties:
         "$ref": AirbyteInternal.yaml
       remoteRegistries:
         "$ref": RemoteRegistries.yaml
+      supportsRefreshes:
```

Comment:
This is the main change

Reply:
If you want this to appear in the registry you will have to add it to ConnectorRegistrySourceDefinition.yaml and ConnectorRegistryDestinationDefinition.yaml as well with defaults

Reply:
Done, I only added it to the destination since it is not needed for the source

Reply:
Is there some converter that are needed as well?

Reply:
Nice, basically any metadata fields will be copied over to the registry so long they are defined in both